plugins/spi/SPIOutput.cpp

```diff
@@ -198,7 +198,8 @@ SPIOutput::SPIOutput(const UID &uid, SPIBackendInterface *backend,
 
   ola::rdm::SlotDataCollection::SlotDataList sd_irgb_combined;
   sd_irgb_combined.push_back(
-      ola::rdm::SlotData::PrimarySlot(ola::rdm::SD_INTENSITY, 0));
+      ola::rdm::SlotData::PrimarySlot(ola::rdm::SD_INTENSITY,
+                                      ola::rdm::DMX_MAX_SLOT_VALUE));
```

Comment:
The tests, but curiously only the Linux ones, not OS X, are failing on this line, see https://travis-ci.org/OpenLightingProject/ola/jobs/345754263#L3298

You want ola::DMX_MAX_SLOT_VALUE, or probably just DMX_MAX_SLOT_VALUE with no namespace.

Reply:
sorry - did not run the compile&test (was late and iam tired and thought - hey just do it quick ;-) )
i fix it and this time first do the compiles myself before i commit ;-)

Reply:
That's fine, that's exactly what the Continuous Integration is there for, we all make mistakes; I certainly do it often enough!